### DISCUSSION TITLE
VehicleManager.SpawnVehicleV3 should use a class or struct before it is too late.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4534] [Ideas]
This was originally brought up by BlazingFlame (Daniel). We feel that if you are already on version three of the method then it would be wise to switch the parameters to a class.
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/1165647a-da70-4ba5-a3f3-d97aea1373db)

**Comment by SDGNelson** [comment]
True, that one is pretty bad - it would certainly benefit from a parameters struct akin to spawn effect or damage methods. "before it is too late" is a little dramatic though as it's not going to explode! ;D

**Reply by Jdance-Media** [reply]
Eh, Blaze's words not mine lol. He phrased it that way, as it would definitely be best to implement it that way before it gets pushed to the main branch.
